ansible/roles/test/files/helpers/pfctest.py

```diff
@@ -0,0 +1,306 @@
+#!/usr/bin/env python
```

Comment:
Script was taken from another repo. Can you please check if licenses are compatible?

Reply:
Sure. I will make sure licenses before checking in.